docs/installing-northstar/troubleshooting.md

```diff
@@ -198,6 +198,16 @@ The following command will reset all your loadouts and levels!
 
 Open console in-game in main menu, type in `ns_resetpersistence` and press enter. Close console again and click on "Launch Northstar". All your stuff should now be reset.
 
+## Player Not Found/Invalid Master Server Token
+
+This is an error commonly caused by EA not properly updating players' names when launching Northstar, especially prevalent for people who have changed their EA username before.
+
+The current solution to this is signing out of the EA App, then _without signing back into the EA App_,open __Vanilla__ Titanfall 2. This should prompt EA App to pop up and ask you to sign in again, sign in on that prompt. Close Vanilla Titanfall 2 and open __Northstar__, and the error should be fixed.
```

Comment:
Double underscore i.e. `__Vanilla__` doesn't exist in standard markdown. Not sure if GitBook supports it for underline like Discord does.

Alternatively use single underscore for `_cursive_` and/or double start for `**bold**` ^^

Reply:
Addressed in 0060569f4ce004fbb36849e5024f0456f1b31e73